openapi/src/parts/azure_landing_zone.yaml

```diff
@@ -27,14 +27,14 @@ paths:
     parameters:
       - $ref: '#/components/parameters/JobId'
     get:
-      summary: Get the result of a async job to create an Azure Landing Zone
+      summary: Get the status of a async job to create an Azure Landing Zone
       operationId: getCreateAzureLandingZoneResult
       tags: [ LandingZones ]
       responses:
         '200':
-          $ref: '#/components/responses/AzureLandingZoneResponse'

```

Comment:
@giventocode I am not seeing this change in Swagger nor when I import the new client (I do see the above change for `CreateLandingZoneResponse`).

![image](https://user-images.githubusercontent.com/484484/200002351-7372a389-852b-4750-bcb7-87398e8246a2.png)